testcases/lib/test_net.sh

```diff
@@ -398,7 +398,7 @@ tst_ping()
 	PING_MAX="${PING_MAX:-500}"
 
 	local src_iface="${1:-$(tst_iface)}"
-	local dst_addr="${2:-$(tst_ipaddr rhost)}"; shift 2
+	local dst_addr="${2:-$(tst_ipaddr rhost)}"; shift $(( $# > 2 ? 2 : 0 ))
```

Comment:
Sorry for my stupid mistake. lol
I also split the following assignment to two line.
This assignment failed and show "local: X: bad variable name" when running testcases.
It may be bug of sh?